pkg/kubelet/status/status_manager.go

```diff
@@ -426,7 +426,7 @@ func (m *manager) updateStatusInternal(pod *v1.Pod, status v1.PodStatus, forceUp
 	// The intent here is to prevent concurrent updates to a pod's status from
 	// clobbering each other so the phase of a pod progresses monotonically.
 	if isCached && isPodStatusByKubeletEqual(&cachedStatus.status, &status) && !forceUpdate {
-		klog.V(3).Infof("Ignoring same status for pod %q, status: %+v", format.Pod(pod), status)
+		klog.V(3).Infof("Ignoring same status for pod %q, status.Phase: %s", format.Pod(pod), status.Phase)
```

Comment:
one more place in this file: https://github.com/kubernetes/kubernetes/blob/9a497c8be6edf04d3b406fd4015f375be066c2ae/pkg/kubelet/status/status_manager.go#L166-L167

Reply:
And here:

https://github.com/kubernetes/kubernetes/blob/9a497c8be6edf04d3b406fd4015f375be066c2ae/pkg/kubelet/status/status_manager.go#L443-L451